src/System.Private.Xml/src/System/Xml/Dom/XmlDocument.cs

```diff
@@ -22,7 +22,7 @@ public class XmlDocument : XmlNode
         private DomNameTable _domNameTable; // hash table of XmlName
         private XmlLinkedNode _lastChild;
         private XmlNamedNodeMap _entities;
-		// TODO: _htElementIdMap can be Dictionary<string, List<WeakReference<XmlElement>>>
+        // TODO: _htElementIdMap can be Dictionary<string, List<WeakReference<XmlElement>>>
```

Comment:
> None of the others seem worthy of an issue. Most look like notes from years ago.

Can we delete them then?  TODOs should represent things to do, and things to do should have issues.  If there's nothing to do, we should be able to delete the TODO, or at the very least change it to a normal comment if it's just descriptive.